man-roxygen/exe.R

```diff
@@ -0,0 +1,2 @@
+#' @param exe Executable name, optionally including the path to the
```

Comment:
maybe capitalize `PATH` to distinguish it better just any folder path. This sentence is a bit confusing as written currently.

Reply:
Good idea to revise this.